.eslintrc.json

```diff
@@ -10,5 +10,8 @@
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "prettier"
-  ]
+  ],
+  "rules": {
```

Comment:
Why do we need global `eslint` and `eslintignore`?

Reply:
In `sdk-node` we have it because we don't have the per package config.
Here, I think it'd make sense to actually run eslint per-package, otherwise what's the point of having those configs?

Reply:
I know, my point is that we should lint with the per-package config so users aren't surprised if they try to lint our code and and it fails.